packages/usa-step-indicator/src/usa-step-indicator.twig

```diff
@@ -1,4 +1,4 @@
-<div class="usa-step-indicator {{ modifier }}" aria-label="progress">
+<div class="usa-step-indicator {{ modifier }}">
```

Comment:
**Note (non-blocking):** I noticed that my screen reader no longer focuses the entire time picker "group".

Before:

1. Screen reader focuses entire component group
2. Screen reader then focuses step indicator bar and reads through steps
3. Screen reader then reads out header element

After:

1. Screen reader focuses step indicator bar and reads through steps
2. Screen reader then focuses header element

This doesn't seem to be an issue but wanted to flag since the behavior changes slightly

Reply:
Thanks for flagging!

@amycole501 and @alex-hull, would this change in behavior count as an accessibility regression? Wanting to check in with you both before moving forward with this.

Reply:
I agree with Charlie. I don't think this will be an issue. It appears that the flow is not that much different between the before and after interaction, so that should not be too big of a change. I also think as long as the user hears what current step they are on, this should not be an issue. @amyleadem